aioshelly/__init__.py

```diff
@@ -229,11 +237,21 @@ def shutdown(self):
         self._update_listener = None
         self._unsub_listening()
 
+    async def _async_init(self):
+        """Async init upon CoAP message event."""
+        try:
+            async with async_timeout.timeout(10):
```

Comment:
I would suggest using a general constant so that if we need it in other places in future we are sure to use the same value